semantic_conventions/trace/general.yaml

```
@@ -48,31 +42,67 @@ groups:
       - id: peer.ip
         type: string
         brief: >
-          Remote address of the peer (dotted decimal for IPv4 or
+          Remote address of the logical peer (dotted decimal for IPv4 or
           [RFC5952](https://tools.ietf.org/html/rfc5952) for IPv6)
         examples: '127.0.0.1'
-      - id: peer.port
+      - id: sock.peer.name
+        type: string
+        brief: Remote socket peer name.
+        examples: proxy.example.com
+      - id: sock.peer.addr
+        type: string
+        brief: >
+          Remote socket peer address (IPv4 or IPv6 for internet protocols, path for local communication,
+           [etc](https://man7.org/linux/man-pages/man7/address_families.7.html)).
+        examples: ['127.0.0.1', '/tmp/mysql.sock' ]
+      - id: sock.peer.port
         type: int
-        brief: 'Remote port number.'
-        examples: [80, 8080, 443]
+        brief: Remote socket peer port (if defined for the address family).
+        examples: 16456
+      - id: sock.family
+        type:
+          allow_custom_values: true
+          members:
+            - id: inet
+              value: 'AF_INET'
```

```suggestion
              value: 'inet'
```

If we want to reference the Unix C define name, we can add it to brief.